src/main/java/edu/cornell/kfs/module/purap/document/authorization/IWantDocumentAuthorizer.java

```diff
@@ -1,24 +1,39 @@
 package edu.cornell.kfs.module.purap.document.authorization;
 
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
 import java.util.Map;
 import java.util.Set;
 
+import org.apache.commons.collections4.CollectionUtils;
+import org.kuali.kfs.kew.actionrequest.ActionRequest;
 import org.kuali.kfs.kew.api.KewApiConstants;
+import org.kuali.kfs.kew.api.WorkflowDocument;
+import org.kuali.kfs.kew.api.document.WorkflowDocumentService;
 import org.kuali.kfs.kim.api.KimConstants;
+import org.kuali.kfs.kim.api.role.RoleService;
+import org.kuali.kfs.kim.api.services.KimApiServiceLocator;
 import org.kuali.kfs.kim.bo.impl.KimAttributes;
 import org.kuali.kfs.kim.impl.identity.Person;
 import org.kuali.kfs.krad.document.Document;
+import org.kuali.kfs.krad.util.GlobalVariables;
```

Comment:
Is this import being used anymore by this class? Can it be removed?

Reply:
Removed